cassdegrees/templates/createcourse.html

```diff
@@ -1,9 +1,20 @@
 {% extends "base.html" %}
+{% load breadcrumbs %}
 
 {% block page-title %}{% if edit %}Edit{% else %}Create{% endif %} Course{% endblock %}
 
 {% block subtitle %}{% if edit %}Edit{% else %}Create{% endif %} Course{% endblock %}
 
+{% block breadcrumb-trail %}
+    {% breadcrumb  '/list/?view=Course' 'Template/Subplan/Course' %}
```

Comment:
`Templates/...`
IMO this should be plural

Reply:
Actually this was done deliberately because I figured it made sense in terms to go `Program >> Add` instead of `Programs >> Add`. Happy to change it if you really wanted to but I wasn't sure which was better

Reply:
I decided to add your changes after consultation with Daniel